_includes/_code/tapbox-c/main.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+//---------------------------- ParentWidget ----------------------------
+
+class ParentWidget extends StatefulWidget {
+  @override
```

Comment:
Why do you use `@override` in some places but not all?  We should either consistently use it everywhere it is allowed or not use it in these examples.